internal/js_library/js_library.bzl

```diff
@@ -54,6 +54,10 @@ _ATTRS = {
         They will be copied into the package bin folder if needed.""",
         allow_files = True,
     ),
+    "npm_package": attr.bool(
+        doc = """If set to true, js_library will provide NpmPackageInfo.""",
```

Comment:
maybe more text here - users should basically never set this unless making their own user-managed third-party dependencies?

Reply:
Added more